onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -667,10 +667,12 @@ def aten_cartesian_prod(tensors: Sequence[TensorType]) -> TensorType:
     raise NotImplementedError()
 
 
+@torch_op("aten::cat")
 def aten_cat(tensors: Sequence[TensorType], dim: int = 0) -> TensorType:
     # cat(Tensor[] tensors, int dim=0) -> Tensor
+    # TODO: onnxscript cannot support parsing correctly input as Tensor[] now
 
-    raise NotImplementedError()
+    return op.Concat(tensors, axis=dim)
```

Comment:
This should probably be `op.ConcatFromSequence`. I assume that `tensors` is an input, not an attribute? That is, the tensors to be concatenated are runtime values right, not statically known constants?